### PR TITLE
Bertec GUI

### PR DESCRIPTION
**Pull request by Katharine-Walters**
Removed logger from BertecGUI.py, now relying on separate logger.py from OSL library. 
Added rounding correction to belt speed to correct for non-constant belt speed reading. 
Added some documentation, changed some variable names for clarity. 
Tested on the Bertec for level ground in forward and backward directions.

### CODE REVIEW

File: BertecGUI.py [hunk 1]
```diff
@@ -9,18 +9,15 @@
 from datetime import datetime
 from BertecMan import Bertec
 from ViconMan import Vicon
-
-from typing import Any, Callable, List, Union
-import csv
-import logging
-from logging.handlers import RotatingFileHandler
+from logger import Logger
```
Comment (tkevinbest) [inline BertecGUI.py:12]: Shouldn't this be `from opensourceleg.utilities.logger import Logger`? Where is the logger.py file?

Comment (Katharine-Walters) [reply BertecGUI.py:12]: I downloaded the file and put it in the same folder. I couldn't pip install the opensourceleg library, and you also have the logger.py file in your folder with BertecGUI

Comment (tkevinbest) [reply BertecGUI.py:12]: Why couldn't you pip install it? The reason I'm being picky here is its two points of maintenance and we can end up with an "old" verison of logger.py. I think we should not have logger.py at all in this repo and just reference the osl library one.

**Comment by tkevinbest** [comment]
#7

**Comment by Katharine-Walters** [comment]
The python version was incorrect. Unfortunately the default python version is 3.8 so I have to re-set the version every time I open VSCode to run it with the OSL library. I made the necessary changes to make the BertecGUI run with the OSL logger and tested with the Bertec.

**Review by tkevinbest — APPROVED** [review]
Looks good!

**Pull request merged**